stdlib/2and3/mmap.pyi

```diff
@@ -71,6 +70,9 @@ if sys.version_info >= (3,):
         def __setitem__(self, index: int, object: int) -> None: ...
         @overload
         def __setitem__(self, index: slice, object: bytes) -> None: ...
+        # Doesn't actually exist, but the object is actually iterable because it has __getitem__ and
+        # __len__, so we claim that there is also an __iter__ to help type checkers.
```

Comment:
I think this is debatable. Maybe it would be better to solve that (eventually) using protocols and for now not make this iterable? Or do you have specific code (maybe priorietary) that depends on it being iterable?